tests/acceptance/smoke-test-slow.js

```diff
@@ -82,6 +82,19 @@ describe('Acceptance: smoke-test', function() {
       });
   });
 
+  it('ember test exits with zero when tests pass', function() {
+    return copyFixtureFiles('smoke-tests/passing-test')
+      .then(function() {
+        return runCommand(path.join('.', 'node_modules', 'ember-cli', 'bin', 'ember'), 'test')
```

Comment:
@kellyselden @stefanpenner what does that entail?

@stefanpenner what time is it?!